spec/controllers/contracts_controller_spec.rb

```diff
@@ -9,9 +9,43 @@
 #  updated_at          :datetime         not null
 #
 
-require 'rails_helper' 
+require 'rails_helper'
 
 RSpec.describe ContractsController, type: :controller do
+  describe 'GET #index' do
```

Comment:
While you are testing that the controller renders the correct view, you aren't testing that your instance variable are being created the way you think they are. We'll want to add some tests for this.